doc/sphinx/Pacemaker_Administration/upgrading.rst

```diff
@@ -203,7 +205,26 @@ while upgrading a cluster node.
      - Pacemaker 1.1.11 [#]_
    * - Pacemaker 1.y.z
      - Pacemaker 1.0.0
-   * - Pacemaker 0.7.z
+   * - Pacemaker 0.6.z to 0.7.z
+     - Pacemaker 0.6.z
+
+The following table lists compatible versions for other nodes in the cluster
+while upgrading a Pacemaker Remote node.
+
+.. list-table:: **Version Compatibility Table for Pacemaker Remote Nodes**
+   :class: longtable
+   :widths: 1 1
+   :header-rows: 1
```

Comment:
I'd make it clearer that the software on the remote node itself follows the same table as cluster nodes. The headings could be clearer here that the first column is the version on the Pacemaker Remote node and the second on the cluster nodes.

Pacemaker Remote was introduced in 1.1.9, so that would be the oldest on either side. (I don't think it was actually usable until 1.1.12 or so, but that's not terribly relevant for upgrading purposes. Without painful tracing and experimentation, I wouldn't be sure what the actual first truly compatible release is, but for documentation purposes 1.1.9 is fine.)

Pacemaker Remote: 1.1.9 to 2.1.x (with an asterisk for 1.1.15 to 1.1.17, which require 1.1.15 or later cluster nodes)
Cluster nodes: 1.1.9 or later

Pacemaker Remote: 3.0.0 or later
Cluster nodes: 2.0.0 or later

Reply:
Good point. Updating commit to address this. Feel free to edit yourself if you'd like

Reply:
Seeing the change, I realized the original table was correct as-is. There are no compatibility concerns when upgrading Pacemaker Remote on a node. That doesn't need to hold up this PR though.

Reply:
> There are no compatibility concerns when upgrading Pacemaker Remote on a node

Huh. Pretty much, although there are minor differences.

* In theory 2.y.z Pacemaker Remote nodes are compatible with 1.1.9 cluster nodes. For cluster nodes, 2.y.z requires 1.1.11 on all other nodes.
* The 1.1.15 bug

Reply:
I mean there are no concerns about upgrading one Pacemaker Remote node relative to the versions of other Pacemaker Remote nodes (the first table). The second table has the concerns relative to cluster nodes.

Reply:
Ah I see